README.md

```diff
@@ -247,6 +247,20 @@ Ecto.Multi.new()
 
 As you can see, the `Carbonite.Transaction` is a great place to store metadata for the operation. A `type` field can be used to categorize the transactions. A `user_id` would be a good candidate for a transaction log, as well.
 
+### Without Ecto.Multi 
+
+If you have a situation, where you want to insert a transaction without `Ecto.Multi`, then you can use `Carbonite.insert_transation/3`. Although grouping multiple operations into one transaction fits the use-case of `Ecto.Multi` perfectly so `Carbonite.Multi.insert_transaction/2` is still the preferred way here.
```

Comment:
```suggestion
If you have a situation where you want to insert a transaction without `Ecto.Multi`, you can use `Carbonite.insert_transation/3`.
```

IMO this sentence about the multi being the preferred way: :shrug: it's mentioned first, guess that's enough?